src/meta/src/stream/stream_manager.rs

```diff
@@ -64,6 +64,11 @@ pub struct CreateMaterializedViewContext {
     pub table_id_offset: u32,
     /// Internal TableID to Table mapping
     pub internal_table_id_map: HashMap<u32, Option<Table>>,
+    /// The set of fragments with their upstream table ids, these fragments need to be colocated
+    /// with their upstream tables. Specifically, they are fragments containing chain nodes.
+    ///
+    /// They are scheduled in `resolve_chain_node`.
+    pub fragment_upstream_table_map: HashMap<FragmentId, TableId>,
```

Comment:
We may use a more specific name for this?

```suggestion
    /// The upstream tables of all fragments containing chain nodes.
    /// These fragments need to be colocated with their upstream tables.
    ///
    /// They are scheduled in `resolve_chain_node`.
    pub fragment_upstream_table_map: HashMap<FragmentId, TableId>,
```

Reply:
🤔 well, you got a better naming?